app/tests/workstation_config_tests/test_views.py

```diff
@@ -58,6 +58,7 @@ def test_workstation_config_create_view(client):
         method=client.post,
         data={"title": "foo"},
     )
+    assert response.status_code == 200
```

Comment:
Status code 200 means that there is an error in the form, it should 302 redirect on creation. You can inspect the errors with something like `response.context["form"].errors`.

Reply:
`{'ghosting_slice_depth': ['This field is required.']}` => You've set this as required in the model (e.g. `blank=False`), which means that you need to send along the data in the form.